app/Http/Requests/AdminRequest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Http\Client\Response;
+use Illuminate\Http\Request;
+
+class AdminRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, mixed>
+     */
+    public function rules(Request $request)
+    {
+        if($request->is('api/admin/login')){
```

Comment:
@Babita3331 Please maintain the separation of concerns. If your different payload with validation rules for  routes, just have different form request files for them